iris/tests/web/routers/test_pipelines.py

```
@@ -0,0 +1,248 @@
+    started = threading.Barrier(3, timeout=0.5)
+
+    def slow_task():
+        started.wait()
+
+    try:
+        futures = [executor.submit(slow_task) for _ in range(3)]
+        with_timeout_errors = 0
+        for f in futures:
+            try:
+                f.result(timeout=1)
+            except Exception:  # pylint: disable=broad-except
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)